apps/api/test/handles.controller.e2e-spec.ts

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable import/no-extraneous-dependencies */
+/* eslint-disable no-undef */
+import { INestApplication, ValidationPipe } from '@nestjs/common';
+import { Test, TestingModule } from '@nestjs/testing';
+import { EventEmitter2 } from '@nestjs/event-emitter';
+import { ApiModule } from '../src/api.module';
+import request from 'supertest';
+
+describe('Handles Controller', () => {
+  let app: INestApplication;
+  let module: TestingModule;
+  beforeEach(async () => {
+    module = await Test.createTestingModule({
+      imports: [ApiModule],
+    }).compile();
+
+    app = module.createNestApplication();
+    const eventEmitter = app.get<EventEmitter2>(EventEmitter2);
+    eventEmitter.on('shutdown', async () => {
+      await app.close();
+    });
+    app.useGlobalPipes(new ValidationPipe());
+    app.enableShutdownHooks();
+    await app.init();
+  });
+
+  it('(POST) /handles creates new account', async () => {
+    await request(app.getHttpServer())
+      .post('/handles')
+      .expect(200)
+      .expect((req) => req.text === 'Account created successfully');
+  });
+
+  it('(GET) /handles/:msaId with valid msaId', async () => {
+    const validMsaId = '1';
+    await request(app.getHttpServer())
+      .get('/handles/' + validMsaId)
```

Comment:
nit: ESLint rule for prefer-template literals instead of string concatenation
```suggestion
      .get(`/handles/${validMsaId}`)
```